tests/tasks/setup_candlepin.yml

```diff
@@ -68,6 +68,9 @@
       loop:
         - "5050"
 
+    - name: Install key for RPM repositories
+      shell: curl http://127.0.0.1:8080/RPM-GPG-KEY-candlepin > /etc/pki/rpm-gpg/RPM-GPG-KEY-candlepin
```

Comment:
can you use the `get_url` module for this instead of `curl`?  https://docs.ansible.com/ansible/2.9/modules/get_url_module.html#get-url-module